pkg/notes/options/options.go

```diff
@@ -184,44 +181,14 @@ func (o *Options) ValidateAndFinish() (err error) {
 	}
 
 	// Set the format
-	format, err := o.template()
-	if err != nil {
-		return err
+	// TODO: Remove "markdown" after some time as it is deprecated in PR#1008
+	if o.Format == "markdown" || o.Format == "" {
+		o.Format = "go-template:default"
```

Comment:
I suggest in making `"go-template:default"` a public const.

Reply:
Done. And also made "markdown" and "json" and "" const as well.